locales/sq_AL/LC_MESSAGES/duckduckgo.po

```diff
@@ -118,24 +118,24 @@ msgstr ""
 #. We realise this is the incorrect way to handle plurals. Sorry, we are working on a fix.
 msgctxt "published dates for organic results"
 msgid "1 day ago"
-msgstr ""
+msgstr "1 ditë përpara"
 
 #. We realise this is the incorrect way to handle plurals. Sorry, we are working on a fix.
 msgctxt "published dates for organic results"
 msgid "1 hour ago"
-msgstr ""
+msgstr "1 orë përpara"
 
 msgctxt "install-extension"
 msgid "1. Open %sDownloads%s"
-msgstr ""
+msgstr "Hap"
```

Comment:
This should be the translation for `1. Open %sDownloads%s`. Please check this again and keep the `1.` at the start. Thank you! 💪